docs/xff-proposal.md

```diff
@@ -0,0 +1,75 @@
+# Istio v1alpha1 proposal
+
+## Summary 
+
+This document proposes  istio-manager API exposed to users. Initial version covers hpa and resources configuration for Istio components such as istiod, ingress gateway and sidecars. This version also covers configuration needed to forward client information to workloads (client address via xff).
+
+## Details
+
+API specification on root level was split into `controlPlane` and `dataPlane` as described in Istio overview. `controlPlane` holds resources and hpa configuration for `istiod` component and general configuration called `meshConfig`. For this moment `meshConfig` will expose configuration to define `numTrustedProxies`, which is required for xff. `dataPlane` configuration contains resources and hpa configuration for `ingressGateway` component.
+
+Status will be implemented per `kyma-operator` requirements.
+
+## Istio CR example
+
+```
+apiVersion: operator.kyma-project.io/v1alpha1
+kind: Istio
+metadata:
+spec:
+  controlplane:
```

Comment:
I think we need to adapt the fields "controlplane" and "dataplane" to use the correct camelCase form, as in the details above.